### DISCUSSION TITLE
国内github下载速度太慢，需求一个国内的镜像！！！

### DISCUSSION THREAD

**Discussion by kamuitlw4058** [BoomingTech/Piccolo#151] [Ideas]
**Is your feature request related to a problem? Please describe. **

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by K1ngst0m** [comment]
https://doc.fastgit.org/zh-cn/guide.html

**Comment by hanlanyi** [comment]
https://github.com/fhefh2015/Fast-GitHub

**Comment by ambiguouStr** [comment]
Maybe there truly is a need to host a mirror at some codehub sites behind the GFW.
